### DISCUSSION TITLE
Image running away from the screen

### DISCUSSION THREAD

**Discussion by dhnsilva** [KlipperScreen/KlipperScreen#1101] [Q&A] [answered]
Hi all!
Does anyone have any problems in klipperscreen related to the image shown on the lcd going off screen gradually?

![IMG_0014](https://github.com/KlipperScreen/KlipperScreen/assets/16030426/a20feff0-c5dc-4f15-946e-326b10becb8f)

**Comment by alfrix** [comment]
the screen resolution seems to be too low, resolutions less than 480x320 are not supported in the official repo, sorry. https://klipperscreen.readthedocs.io/en/latest/Hardware/

i have planned to reduce the size of scrollbar a little bit, and that can help, but don't take it as support, nothing is designed for less than 480x320

**Reply by dhnsilva** [reply]
The resolution is 480x320... is a MKS screen proper to Klipperscreen. 
The screens begin well but when we interact with it the the image enlarge and coming to the left, is necessary reload the Klipperscreen (changing the font size for example) to come at correct position.

**Reply by alfrix** [reply]
> when we interact with it the the image enlarge and coming to the left

i will need the full logfile to continue (https://klipperscreen.readthedocs.io/en/latest/Troubleshooting/)

**Reply by dhnsilva** [reply]
[KlipperScreen.log](https://github.com/KlipperScreen/KlipperScreen/files/12469913/KlipperScreen.log)

**Comment by alfrix** [comment]
i've found this to be an issue, i will fix it in the weekend

**Comment by alfrix** [comment, marked as answer]
closing as completed with e80ef7e
